### ISSUE TITLE
The MMKV for HarmonyOS NEXT version lacks the interface for parsing the float type.

### ISSUE THREAD

**Issue by xiaoxizi2303** [Tencent/MMKV#1280]
### The version of MMKV
>v1.3.5

### What's the issue?
> The MMKV for HarmonyOS NEXT version lacks the interface for parsing the float type.

**Comment by xiaoxizi2303** [comment]
I need to parse the mmkv storage file generated in the Android project on the Harmony project. This file stores data of the float type. Therefore, an interface of the float type is required to parse the data of the float type.

**Comment by lingol** [comment]
Noted.

**Comment by xiaoxizi2303** [comment]
Thank you. When can this interface be provided? Because the customer encounters the problem that the float type cannot be parsed when using the mmkv library.

**Comment by lingol** [comment]
Yes.

**Comment by xiaoxizi2303** [comment]
34/5000 拼音  翻译
Hello, when can the modification of this problem be released to the ohpm central warehouse of Harmony?

**Comment by xiaoxizi2303** [comment]
Hello, when can the modification of this problem be released to the ohpm central warehouse of Harmony?

**Comment by lingol** [comment]
> Hello, when can the modification of this problem be released to the ohpm central warehouse of Harmony?

If you are in a hurry, we can publish an alpha version of 1.3.6-alpha.

**Comment by xiaoxizi2303** [comment]
Yes, we really need this version now, thank you very much

**Comment by lingol** [comment]
v1.3.6-alpha is live on OHPM.

**Comment by xiaoxizi2303** [comment]
Okay, thank you very much.

**Comment by HandwerSTD** [comment]
After upgrading to 1.3.6-alpha I found some warnings occurred when compiling. However mmkv still works fine so this might not be a bug, but it's still a bit annoying. 

```plain
> hvigor WARN: ArkTS:WARN File: D:/DevEcoProjects/SomeProject/oh_modules/.ohpm/@tencent+mmkv@1.3.6-alpha/oh_modules/@tencent/mmkv/src/main/ets/utils/MMKV.ets:119:13
 Use explicit types instead of "any", "unknown" (arkts-no-any-unknown)

WARN: ArkTS:WARN File: D:/DevEcoProjects/SomeProject/oh_modules/.ohpm/@tencent+mmkv@1.3.6-alpha/oh_modules/@tencent/mmkv/src/main/ets/utils/MMKV.ets:139:13
 Use explicit types instead of "any", "unknown" (arkts-no-any-unknown)

WARN: ArkTS:WARN File: D:/DevEcoProjects/SomeProject/oh_modules/.ohpm/@tencent+mmkv@1.3.6-alpha/oh_modules/@tencent/mmkv/src/main/ets/utils/MMKV.ets:735:13
 Use explicit types instead of "any", "unknown" (arkts-no-any-unknown)

WARN: ArkTS:WARN File: D:/DevEcoProjects/SomeProject/oh_modules/.ohpm/@tencent+mmkv@1.3.6-alpha/oh_modules/@tencent/mmkv/src/main/ets/utils/MMKV.ets:752:13
 Use explicit types instead of "any", "unknown" (arkts-no-any-unknown)

WARN: ArkTS:WARN: For details about ArkTS syntax errors, see FAQs
WARN: ArkTS:WARN File: D:/DevEcoProjects/SomeProject/oh_modules/.ohpm/@tencent+mmkv@1.3.6-alpha/oh_modules/@tencent/mmkv/src/main/ets/utils/MMKV.ets:21:20
 Currently module for 'libmmkv.so' is not verified. If you're importing napi, its verification will be enabled in later SDK version. Please make sure the corresponding .d.ts file is provided and the napis are correctly declared.
```

**Comment by lingol** [comment]
Noted.

**Comment by xiaoxizi2303** [comment]
This problem is not found in the Harmony project of the new version.

**Comment by lingol** [comment]
All except this one are fixed. I can't find any document about this warning.
>Currently module for 'libmmkv.so' is not verified. If you're importing napi, its verification will be enabled in later SDK version. Please make sure the corresponding .d.ts file is provided and the napis are correctly declared.

**Comment by xiaoxizi2303** [comment]
Other alarms are generated because the Harmony SDK updates the rules for verifying variable types.

**Comment by lingol** [comment]
The warning is fixed in v1.3.6-beta on OHPM.

**Comment by xiaoxizi2303** [comment]
Okay, thanks.

**Comment by lingol** [comment]
It's fixed in [v1.3.6](https://github.com/Tencent/MMKV/releases/tag/v1.3.6).
